lib/elixir/lib/module.ex

```diff
@@ -861,6 +861,44 @@ defmodule Module do
     end, tuples)
   end
 
+  @spec make_overridable(module, module) :: :ok
+  def make_overridable(module, behaviour) when is_atom(module) and is_atom(behaviour) do
+    case check_module_for_overridable(module, behaviour) do
+      :ok -> :ok
+      {:error, error_explanation} ->
+        raise ArgumentError, "cannot pass module #{inspect(behaviour)} as argument to defoverridable/1 because #{error_explanation}"
+    end
+
+    behaviour_callbacks = :lists.map(&normalize_macro_or_function_callback/1, behaviour.behaviour_info(:callbacks))
+
+    tuples = for function_tuple <- definitions_in(module),
+                 function_tuple in behaviour_callbacks,
+             do: function_tuple
```

Comment:
It doesn't implement any specific style guide, and development largely tailed off when i joined a crazy start up a couple years back.

Credo seems like a good tool though I've not used it. I also don't know if it aligns with the style used in this code base.

Generally I'm more interested in formatters these days, but that's another matter we can chat about elsewhere :)